evm_loader/program/src/executor/precompile_extension/spl_token.rs

```diff
@@ -695,9 +656,11 @@ fn get_account<B: AccountStorage>(
 ) -> Result<Vec<u8>> {
     let account = state.external_account(account)?;
     let token = if spl_token::check_id(&account.owner) {
-        spl_token::state::Account::unpack_unchecked(&account.data)?
-    } else {
+        spl_token::state::Account::unpack(&account.data)?
+    } else if system_program::check_id(&account.owner) {
         spl_token::state::Account::default()
```

Comment:
In `get_account` you return a default values if account is owned by system program.
But in `get_mint` you return an error.

Keep it consistent.

Reply:
Fixed